change/@ni-nimble-components-c29f703d-f01e-45c4-942f-c887470025b8.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
> Do we need to update any of those?

Mostly, no, because their role or other ARIA attributes (e.g. `aria-expanded`, `aria-checked`) already communicate the meaning of the icons. I was on the fence but decided to add `role` and `aria-label` to the banner severity glyphs.

> Can we document this guidance in CONTRIBUTING.md or accessibility.md?
Added to accessibility.md.